### ISSUE TITLE
confusing message from configure when python is not present

### ISSUE THREAD

**Issue by verygreen** [zfsonlinux/zfs#8861]
As I was trying to build zfs 0.8.0 from source on a minimal rhel8 image, I got this nice and descriptive error from configure:

```
configure: error: Cannot find  in your system path
```

In the end I traced it to this bit in config/always-python.m4:
```
AC_MSG_ERROR([Cannot find $PYTHON in your system path])
```

perhaps it's worth adding "python interpreter" in front of $PYTHON similar to what's done in config/ax_python_devel.m4

**Comment by behlendorf** [comment]
This was actually just fixed by 1a132f063882bba08253b6c0abf3846cdacfb7a6 so expect this to be resolve in 0.8.1.

**Comment by verygreen** [comment]
yes, you are correct. I just checked on the latest checkout and no error there,

Thanks!
